File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        
        String[] tokens = my_string.split(" ");
        
        int answer = Integer.parseInt(tokens[0]);
        
        for (int i = 1; i < tokens.length; i += 2) {
            String operator = tokens[i]; 
            int nextNumber = Integer.parseInt(tokens[i + 1]); 
            
            if (operator.equals("+")) {
                answer += nextNumber;
            } else if (operator.equals("-")) {
                answer -= nextNumber;
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    let tokens = my_string.split(" ");
    let answer = parseInt(tokens[0]);
    
    for (let i = 1; i < tokens.length; i += 2) {
        let operator = tokens[i];
        let nextNumber = parseInt(tokens[i + 1]);
        
        if (operator === "+") {
            answer += nextNumber;
        } else if (operator === "-") {
            answer -= nextNumber;
        }
    }
    
    return answer;
}